prospero-cli/src/main/java/org/wildfly/prospero/cli/spi/CliProviderImpl.java

```diff
@@ -53,15 +53,17 @@ public String getApplyUpdateCommand(Path installationPath, Path candidatePath) {
         return CliConstants.Commands.UPDATE + " " + CliConstants.Commands.APPLY + " "
                 + CliConstants.DIR + " " + escape(installationPath.toAbsolutePath()) + " "
                 + CliConstants.CANDIDATE_DIR + " " + escape(candidatePath.toAbsolutePath()) + " "
-                + CliConstants.YES;
+                + CliConstants.YES + " "
+                + CliConstants.REMOVE;
     }
 
     @Override
     public String getApplyRevertCommand(Path installationPath, Path candidatePath) {
         return CliConstants.Commands.REVERT + " " + CliConstants.Commands.APPLY + " "
                 + CliConstants.DIR + " " + escape(installationPath.toAbsolutePath()) + " "
                 + CliConstants.CANDIDATE_DIR + " " + escape(candidatePath.toAbsolutePath()) + " "
-                + CliConstants.YES;
+                + CliConstants.YES + " "
+                + CliConstants.REMOVE;
```

Comment:
should this be controlled by a parameter?

Reply:
I thought about that but ended up it could be the same case as `CliConstants.YES`. In addition, I did not find it useful for now in our current use case. After applying the candidate server, we should delete it.

I don't mind if it gets parametrized, but found it unuseful now, so went with the easiest path.

Reply:
OK makes sense